apps/api/src/server/app/inspector/inspector.controller.js

```diff
@@ -63,6 +63,132 @@ const getAppeals = async function(request, response) {
 	return response.send(appealsForResponse);
 };
 
+const getAppealDetails = async function(request, response) {
+	const appeal = await appealRepository.getByIdIncluding(request.params.appealId, {
+		appellant: true, 
+		validationDecision: {
+			where: {
+				decision: 'complete',
+			},
+		},
+		lpaQuestionnaire: true,
+		appealDetailsFromAppellant: true
+	});
+	const formattedAppeal = {
```

Comment:
Missing:
- appeal age
- address
- local planning department
- booked site visit data

Reply:
aha, this wasnt in the ticket to begin with. thanks for letting me know